ds-prep-python-loops/main.py

```diff
@@ -0,0 +1,33 @@
+record = (1, "Grimdiana", "Bones", "boulders")
+row = ""
+for x in record:
+  print(x)
+for x in record:
+  for y in row:
+    print(x , "," , y)
+values_list = [1, 1, 2, 3, 5, 8, 13, 21, 34]
+for x in range(len(values_list)):
+  print(values_list[x])
+index_list = []
+for x in values_list:
+  for y in index_list:
+    print(x , y)
```

Comment:
Each index of `values_list` should be `prepend`ed into `index_list`, so that `index_list` has all the indexes from `values_list` in reverse order:

---

![image](https://user-images.githubusercontent.com/16143771/149633417-a64e1e70-802c-4914-96e7-01af16cfb160.png)